app/MMB/src/scenes/timeline/Timeline.js

```diff
@@ -32,6 +33,13 @@ function getValidTimelineEvent(data: TimelineEventType, index) {
             <AirportArrivalTimelineEvent data={data} />
           </React.Fragment>
         );
+      case 'LeaveForAirportTimelineEvent':
+        return (
+          <React.Fragment key={'TimelineEvent-' + data.__typename + index}>
+            {data && data.timestamp && <DaySeparator date={data.timestamp} />}
+            <LeaveForAirportTimelineEvent data={data} />
+          </React.Fragment>
```

Comment:
This is clearly quite repetitive. Let's stop here and try to think how to improve it, please...
 🙂

```js
// this is basically still the same
<React.Fragment key={'TimelineEvent-' + data.__typename + index}>
  {data && data.timestamp && <DaySeparator date={data.timestamp} />}
  <Component data={data} />
</React.Fragment>
```

Moreover, I don't think you should add the `DaySeparator` everywhere. This should be done automatically. So maybe it's still just WIP? (but even MVP can be DRY)

Reply:
Okay, I was planning to deal with the DaySeparator later in a different branch, but I guess I'll do it in this one so that next PR's are nicer.

Reply:
No, it's fine - I am OK with this explanation. I just wanted to stop it but since you are going to fix it then we are good to go.